docs/lang/articles/reference.md

```diff
@@ -198,6 +198,24 @@ compile time.
 
 ### Conditional expressions
 
+### Static expressions
+
+```
+static_expression ::= "ti.static(" positional_arguments ")"
+```
+
+Static expressions are expressions that are wrapped by a call to `ti.static()`.
+The `positional_arguments` is evaluated at compile time, and the variables inside it must be compile-time constants.
+
+`ti.static()` receives one or more arguments.
+If one argument it passed to it, it returns the argument.
```

Comment:
Use a bullet list?

* For  a single argument, `ti.static` returns the evaluated argument.
* Otherwise, if more than one arguments are passed in, it returns ...